model-optimizer/extensions/front/YOLO.py

```diff
@@ -40,7 +50,15 @@ class YoloV3RegionAddon(FrontReplacementFromConfigFileGeneral):
     """
     replacement_id = 'TFYOLOV3'
 
+
     def transform_graph(self, graph: Graph, replacement_descriptions):
+        enable_postprocess = replacement_descriptions.get('include_postprocess', False)
+        if enable_postprocess:
+            input_node = Node(graph, 'inputs') # needs to find another way to get input
+            hNorm = input_node.shape[1]
+            wNorm = input_node.shape[2]
+            hw_norm = 32
```

Comment:
Why it is here? We have already a check below

Reply:
But we shouldn't do this code in cycle